src/main/flight/failsafe.c

```diff
@@ -220,5 +285,6 @@ void failsafeUpdateState(void)
  */
 void failsafeOnRxCycleStarted(void)
 {
+    // FIXME Timing is based on the assumption that this function is called at 50Hz
     failsafeState.counter++;
```

Comment:
We *could* address the FIXME as the next failsafe PR, but I'd rather the FIXME was fixed before merging this as it will be cleaner that way around.  Generally speaking bugs should always be fixed before adding new code.  Building on code that has bugs is not good practice, leads to unpredictable behavior and costs more time in the long run.